cnf-tests/Dockerfile

```diff
@@ -30,22 +30,24 @@ WORKDIR $TESTER_PATH
 
 RUN go build -mod=vendor -o /sctptest
 
-# Build oslat-runner binary
-FROM golang:1.13 AS builder-oslat-runner
+# build latency-test's runner binaries
+FROM golang:1.13 AS builder-latency-test-runners
```

Comment:
Can you switch this to golang 1.16 (or above) ?

Reply:
maybe we should do this for all in a different PR

Reply:
+1 for bump to golang 1.16 and doing it in a separate PR

Reply:
So you want me to change it here or on a separate PR?

Reply:
separate PR please